web/public/js/controllers/user-profile-controller.js

```diff
@@ -151,12 +152,13 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   profile.data.dob = new Date(profile.data.dob);
 
   $scope.profile = profile.data;
+  $scope.canEdit = $scope.profile.ownProfileFlag || $scope.profile.myChild;
 
-  cdUsersService.getAvatar($scope.profile.id, function(response){
-    if(!_.isEmpty(response)) {
-      $scope.profile.avatar = 'data:' + response.imageInfo.type + ';base64,' + response.imageData;
-    }
-  });
+  // cdUsersService.getAvatar($scope.profile.id, function(response){
+  //   if(!_.isEmpty(response)) {
+  //     $scope.profile.avatar.data = 'data:' + response.imageInfo.type + ';base64,' + response.imageData;
+  //   }
+  // });
```

Comment:
Must be removed, not necessary anymore since we use the api directly instead of basse64